devicetypes/qubino/qubino-flush-2-relay.src/qubino-flush-2-relay.groovy

```diff
@@ -74,11 +74,11 @@ metadata {
 def installed() {
 	if (zwaveInfo?.model.equals("0051")) {
 		state.numberOfSwitches = 2
-	} else {
+	} else if (zwaveInfo?.model.equals("0052") || zwaveInfo?.model.equals("0053")) {
 		state.numberOfSwitches = 1
 	}
 	
-	if (!childDevices) {
+ 	if (!childDevices && state.numberOfSwitches > 1) {
 		addChildSwitches(state.numberOfSwitches)
```

Comment:
What was happening before with `for (def endpoint : 2..numberOfSwitches) {` when `numberOfSwitches` was 1?

Reply:
It was still adding two children switches